src/iapitest.cpp

```diff
@@ -4,6 +4,7 @@
 #include <irods/irods_client_api_table.hpp>
 #include <irods/irods_pack_table.hpp>
 #include <irods/rodsClient.h>
+#include <irods/rodsError.h>
```

Comment:
Is this needed?

Reply:
Yes. I had inserted a call to `printErrorStack` without including its header.

Reply:
Oh my. The diff outside of including the header didn't show for me the first time for all the places that I made this comment! Weird... Anyways, very good. Resolving.